web_src/js/components/DashboardRepoList.vue

```diff
@@ -431,6 +432,39 @@ const sfc = {
 
     statusColor(status) {
       return commitStatus[status].color;
+    },
+
+    reposFilterKeyControl(e) {
+      switch (e.key) {
+        case 'Enter':
+          document.querySelector('.repo-list-link.active').click();
```

Comment:
If the list is empty (no item), or no active item, this code causes JS error because `document.querySelector('.repo-list-link.active')` returns null

I guess it could be `document.querySelector('.repo-list-link.active')?.click();`